nvidia-nano/GiveMeWES.yaml

```diff
@@ -0,0 +1,597 @@
+
+    - name: Install waggle-node-hostname #the tool is used for a work around to change the hostname 
+      ansible.builtin.apt:
+        deb: https://github.com/waggle-sensor/waggle-node-hostname/releases/download/v1.2.1/waggle-node-hostname_1.2.1_all.deb
+  
+    - name: Install waggle-commontools to read the config
+      ansible.builtin.apt:
+        deb: https://github.com/waggle-sensor/waggle-common-tools/releases/download/v1.0.0/waggle-common-tools_1.0.0_all.deb
+
+    - name: Install Internet sharing service # Install waggle common tools first it is depended on it
+      ansible.builtin.apt:
+        deb: https://github.com/waggle-sensor/waggle-internet-share/releases/download/v1.4.1/waggle-internet-share_1.4.1_all.deb
+
+    - name: Install the registration service
+      ansible.builtin.apt:
+        deb: https://github.com/waggle-sensor/waggle-bk-registration/releases/download/v2.2.2/waggle-bk-registration_2.2.2_all.deb
+
+    - name: Install the reverse tunnel service
+      ansible.builtin.apt:
+        deb: https://github.com/waggle-sensor/waggle-bk-reverse-tunnel/releases/download/v2.3.2/waggle-bk-reverse-tunnel_2.3.2_all.deb
+
+- name: Remove Uneeded Items
+  hosts: all
+  tasks:
+    - name: Remove packages
+      ansible.builtin.apt:
+        name: isc-dhcp-server, whoopsie
+        state: absent
+        purge: yes
+
+    - name: Remove dependencies that are no longer required
+      ansible.builtin.apt:
+        autoremove: yes
+
+    - name: Disable apt-daily service
+      ansible.builtin.systemd:
+        name: apt-daily.service
+        enabled: no
+    
+    - name: Disable apt-daily timer
+      ansible.builtin.systemd:
+        name: apt-daily.timer
+        enabled: no
+
+    - name: Disable apt-daily-upgrade service
+      ansible.builtin.systemd:
+        name: apt-daily-upgrade.service
+        enabled: no 
+
+    - name: Disable apt-daily-upgrade timer
+      ansible.builtin.systemd:
+        name: apt-daily-upgrade.timer
+        enabled: no 
+
+    - name: Remove excutable permissions to update motd directory files
+      ansible.builtin.file:
+        path: /etc/update-motd.d
+        state: directory
+        mode: a-x 
+        recurse: yes
+
+    - name: Add enabled=0 to motd news file
+      shell: echo 'ENABLED=0' > /etc/default/motd-news #could not find module that will replace all file content with new content
+      args:
+        executable: /bin/bash
+
+    - name: Disable motd news service
+      ansible.builtin.systemd:
+        name: motd-news.service
+        enabled: no
+
+    - name: Disable motd news timer
+      ansible.builtin.systemd:
+        name: motd-news.timer
+        enabled: no
+
+- name: Copy ROOTFS into the System
+  hosts: all
+  tasks:
+    - name: Add the WaggleOS MOTD
+      ansible.builtin.copy:
+        src: '{{ playbook_dir }}/ROOTFS/etc/update-motd.d/05-waggle'
```

Comment:
change this to a `with_items` that has permissions and mode in it it, so we can just loop them

Reply:
later update